core/src/main/java/org/infinispan/configuration/parsing/Attribute.java

```diff
@@ -181,4 +181,9 @@ public static Attribute forName(String localName) {
         final Attribute attribute = attributes.get(localName);
         return attribute == null ? UNKNOWN : attribute;
     }
+
+   @Override
+   public String toString() {
+      return name;
```

Comment:
No, it's a quick way to obtain the actual element name without invoking getLocalName()